test/functional/basic/basic-e2e-specs.js

```diff
@@ -203,9 +203,10 @@ describe('XCUITestDriver - basics -', function () {
       await B.promisify(fullImg.parse, {context: fullImg})(fullB64);
       const viewImg = new PNG({filterType: 4});
       await B.promisify(viewImg.parse, {context: viewImg})(viewB64);
-      fullImg.height.should.eql(viewImg.height + Math.round(pixelRatio * statBarHeight));
+      // Viewport size can be smaller than the full image size + status bar on some devices.
+      fullImg.height.should.be.gte(viewImg.height + Math.round(pixelRatio * statBarHeight));
```

Comment:
Okay, I changed the condition https://github.com/appium/appium-xcuitest-driver/pull/1480#issuecomment-1371063721